web/geosearch/datapunt_geosearch/blueprints/search.py

```diff
@@ -76,14 +80,26 @@ def search_everywhere():
         x=x,
         y=y,
         rd=rd,
-        limit=limit
+        limit=limit,
     ))
 
     return Response(generate_async(
-        request_args=request_args
+        request_args=request_args,
+        authz_scopes=getattr(g, "authz_scopes", None)
```

Comment:
For something as important as authorization, perhaps have a global function instead? e.g. `get_current_authz_scopes()` ?

Reply:
Yeah, good idea.